org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JavaClientConnection.java

```diff
@@ -58,6 +58,14 @@
 		@JsonNotification("language/actionableNotification")
 		void sendActionableNotification(ActionableNotification notification);
 
+		/**
+		 * The event notification is sent from a server to a client to tell the 
+		 * client that some certain events happen at server side, like: classpath updated.

```

Comment:
The event notification is sent from a server to a client to notify the client certain events happened on the server side, eg. classpath was updated, projects were imported